src/Crawler.php

```diff
@@ -152,10 +152,26 @@ public function crawlSite( string $static_site_path ) : void {
                     $crawled_contents = (string) $response->getBody();
                     $status_code = $response->getStatusCode();
 
+                    $is_cacheable = true;
                     if ( $status_code === 404 ) {
                         WsLog::l( '404 for URL ' . $root_relative_path );
                         CrawlCache::rmUrl( $root_relative_path );
+                        // Delete crawl queue to prevent crawling not found urls forever.
+                        CrawlQueue::rmUrl( $root_relative_path );
```

Comment:
@utchy I love this style of removing the URLs if they were a 404 in previous crawl!

A concern, though, is in the scenario where a URL was 404 when I developed my site in the morning, due to an error, but then I fix that in the afternoon and run full WP2Static workflow again, will it ever be re-crawled? (I assume "yes", due to re-running `detect` method again, but want to confirm with you, as I've been away from the code for a while 🙇

Reply:
@leonstafford Thanks for your review!

>  will it ever be re-crawled? (I assume "yes", due to re-running `detect` method again

Yes it will be back again in the `detect` process as you mentioned 👍